modules/tinymce/src/core/main/ts/api/util/XHR.ts

```diff
@@ -35,6 +35,9 @@ interface XHR extends Observable<XHREventMap> {
 
 /**
  * This API allows you to send XMLHTTPRequests cross browser. Extends the Observable class.
+ * <br>
+ * <em><code>tinymce.util.XHR</code> has been deprecated and marked for removal in TinyMCE 6.0.</em>
```

Comment:
```suggestion
 * <em><code>XHR</code> has been deprecated and marked for removal in TinyMCE 6.0.</em>
```
I missed this one when removing the namespaces sorry :cry: